tests/apollo/test_skvbc_view_change.py

```diff
@@ -159,8 +159,7 @@ class SkvbcViewChangeTest(unittest.TestCase):
 
             await tracker.tracked_read_your_writes()
 
-            async with trio.open_nursery() as nursery:
-                nursery.start_soon(tracker.run_concurrent_ops, 100)
+            await tracker.run_concurrent_ops(100)
```

Comment:
this "pattern" repeats in several places - can you clean it in the entire codebase?

Reply:
Sure.